lib/appium_lib/ios/xcuitest/command/gestures.rb

```diff
@@ -26,11 +28,12 @@ def swipe(direction:, element: nil)
         #   ```ruby
         #   scroll direction: "down"
         #   ```
-        def scroll(direction:, name: nil, element: nil, to_visible: nil, predicate_string: nil)
+        def scroll(direction:, distance: nil, name: nil, element: nil, to_visible: nil, predicate_string: nil) # rubocop:disable Metrics/ParameterLists
```

Comment:
Rubocop pointed out another issue... Could you fix it like https://github.com/appium/ruby_lib_core/blob/21620568f14ed4537b43c29e2b28026795e30617/lib/appium_lib_core/common/device/image_comparison.rb#L105-L111 ?

You can run rubocop with `rake rubocop` on your local, btw.

thanks for the contribution

Reply:
I modified the line as suggested. I ran the `rake rubocop` I see other warnings as well when I ran the command, should I be worried?

Reply:
ah, yeah. Rubocop raises warnings depends on its versions.
you can run it with `bundle exec rake rubocop` to specify the version.